libs/libc/time/lib_localtime.c

```diff
@@ -2801,6 +2801,11 @@ void tzset(void)
   nxrmutex_unlock(&g_lcl_lock);
 }
 
+void tzreset(void)
+{
+  strlcpy(g_lcl_tzname, "", sizeof(g_lcl_tzname));
```

Comment:
should we add a Kconfig to skip g_lcl_tzname related code?

Reply:
I am not sure but probably no, because localtime would then have to reaload data from zoneinfo file every time `localtime_r()` or  `localtime()` is called (every time you call `date` from NSH, every iteration your app reads time etc.). I think the current implementation in `tzset()` is good.